core/src/main/java/org/elasticsearch/bootstrap/JVMCheck.java

```diff
@@ -118,7 +118,7 @@ static void check() {
             HotspotBug bug = JVM_BROKEN_HOTSPOT_VERSIONS.get(Constants.JVM_VERSION);
             if (bug != null) {
                 if (bug.workAround != null && ManagementFactory.getRuntimeMXBean().getInputArguments().contains(bug.workAround)) {
-                    Loggers.getLogger(JVMCheck.class).warn(bug.getWarningMessage());
+                    Loggers.getLogger(JVMCheck.class).warn("{}", bug.getWarningMessage());
```

Comment:
yeah, there are a few cases where you have to put "{}", but the advantage is that toString() is only evaluated then when we log at that level. 
Related: http://www.slf4j.org/faq.html#string_contents